CHANGELOG.md

```diff
@@ -25,6 +25,56 @@
   characters to the console on Windows. Calls to `Stdout.add*()` behave as
   before.
 
+### Strong Mode
```

Comment:
I think the last few releases has had these under the top `###Language` section, with the individual changes then calling out if they are strong mode specific or not?